tests/tensorflow/quantization/test_algorithm_quantization.py

```diff
@@ -284,12 +284,11 @@ def test_quantize_inputs():
     model, _ = create_compressed_model_and_algo_for_test(model, config, force_no_init=True)
     ref_fake_quantize_layers_for_inputs = {
         'rescaling/fake_quantize',
-        'input_2/fake_quantize',
-        'input_3/fake_quantize',
-        'input_4/fake_quantize',
-        'input_5/fake_quantize'
+        'input_2/fake_quantize/unified_scale_group',
+        'input_4/fake_quantize/unified_scale_group',
+        'input_3/fake_quantize'
     }
-    ref_fake_quantize_layers = 17
+    ref_fake_quantize_layers = 12
```

Comment:
This test case should either use a model where the inputs are not unified by scales, or a 'TRIAL' mode - in essence, an option prohibiting the unified scale invocation - so that the test case is more limited in what it actually requires from NNCF in order to pass.

Reply:
Added 'TRIAL' mode